pyproject.toml

```diff
@@ -82,8 +82,13 @@ include = [
     { path = "sytest-blacklist", format = "sdist" },
     { path = "tests", format = "sdist" },
     { path = "UPGRADE.rst", format = "sdist" },
+    { path = "rust/Cargo.toml", format = "sdist" },
```

Comment:
do we need to include the `Cargo.toml` for the workspace root?
Sounds like maybe we don't *need* to but perhaps we should in case we wind up depending on it for options of some sort?

Reply:
oooh, yes